bundle/Command/Validators.php

```diff
@@ -52,6 +52,19 @@ public static function validateLanguageCode(?string $languageCode = null): strin
         return $languageCode;
     }
 
+    public static function validateDesignType(?string $designType = null): string
+    {
+        if (empty($designType)) {
+            return '';
```

Comment:
Maybe return `local` by default?

Reply:
Added in 065f8190a7d671d06dbf76c7123caa78459e55b6